### PR TITLE
[NO-TASK] Add bs link and price to route details

### PR DESCRIPTION
**Pull request by Kolibri1990**

### CODE REVIEW

File: app/components/RouteHistory.tsx [hunk 1]
```diff
@@ -71,6 +79,12 @@ const RouteHistory = ({ history }: Props) => {
     return formatAmountAndAsset(tx.outputAmount, tx.outputAsset, tx, DirectionType.OUTPUT);
   };
 
+  const priceOf = (tx: RouteHistoryAlias): string => {
+    return formatPrice(tx.inputAmount,tx.outputAmount,tx.inputAsset, tx.outputAsset, tx);
+  };
```
Comment (davidleomay) [inline app/components/RouteHistory.tsx:84]: You use 5 parameters here, but `formatPrice` only has 3? Maybe you wanted to add assets to the formatted price (which would make sense)?

File: app/i18n/es.json [hunk 2]
```diff
@@ -202,7 +202,10 @@
         "ref_volume": "Volumen de referidos",
         "ref_bonus": "Bonificación de referidos",
         "ref_commission": "Comisión de referidos",
-        "change": "Cambiar la dirección"
+        "change": "Cambiar la dirección",
+        "vip_title": "Welcome to the Black Squad",
+        "vip_description": "The Black Squad is an elite group for major investors on DFX and gives you a direct voice channel to DFX leadership.",
+        "vip_link": "The Black Squad gathers in a Telegram group - just follow the link:"
```
Comment (davidleomay) [inline app/i18n/es.json:208]: No translation (also FR, IT, PT)?

File: app/models/User.ts [hunk 3]
```diff
@@ -119,6 +119,7 @@ export interface UserDetailDto extends UserDto {
   buyVolume: VolumeInformation;
   sellVolume: VolumeInformation;
   cryptoVolume: VolumeInformation;
+  bsLink: string;
```
Comment (davidleomay) [inline app/models/User.ts:122]: Missing `?`

File: app/utils/Utils.ts [hunk 4]
```diff
@@ -19,7 +19,8 @@ export const createRules = (rules: any): any => {
   return rules;
 };
 
-export const openUrl = (url: string, newTab = true): void => {
+export const openUrl = (url: string | undefined, newTab = true): void => {
+  if (!url) return;
```
Comment (davidleomay) [inline app/utils/Utils.ts:23]: As far as I can see openUrl is never called with `undefined` (there is a `user.bsLink &&  (...)`)

File: app/screens/home/HomeScreen.tsx [hunk 5]
```diff
@@ -334,6 +334,34 @@ const HomeScreen = ({ session, settings }: { session?: Session; settings?: AppSe
...
+                              <CompactCell multiLine style={{ flex: 3 }}>
+                                {t('model.user.vip_description')}
+                              </CompactCell>
+                            </CompactRow>
+                            <CompactRow>
+                            <CompactCell multiLine style={{ flex: 3 }}>
+                                {t('model.user.vip_link')}
+                              </CompactCell>
+                            <View style={{ flex: 2, flexDirection: "row" }}>
+                                {'open-in-new' && (
```
Comment (davidleomay) [inline app/screens/home/HomeScreen.tsx:354]: What is `open-in-new`?

File: app/screens/home/HomeScreen.tsx [hunk 6]
```diff
@@ -334,6 +334,34 @@ const HomeScreen = ({ session, settings }: { session?: Session; settings?: AppSe
...
+                                  <CompactCell style={{ flex: undefined }}>
+                                    <IconButton icon={'open-in-new'} onPress={device.SM ? () => openUrl(user.bsLink) : undefined} />
+                                  </CompactCell>
+                                )}
+                              </View>
+                              </CompactRow>
+                          </TouchableOpacity>
+                  </DataTable>
+                </>
+              )}
```
Comment (davidleomay) [inline app/screens/home/HomeScreen.tsx:364]: Use auto formatter

File: app/screens/home/HomeScreen.tsx [hunk 7]
```diff
@@ -334,6 +334,34 @@ const HomeScreen = ({ session, settings }: { session?: Session; settings?: AppSe
               </DataTable>
               <SpacerV />
 
+              {user.bsLink && (
+                <>
+                  <SpacerV height={50} />
+                  <H2 text={t("model.user.vip_title")} />
+                  <SpacerV />
+                  <DataTable>
+                          <TouchableOpacity onPress={ () => openUrl(user.bsLink)} key={'model.user.vip_description'} disabled={!'open-in-new'  || device.SM}>
```
Comment (davidleomay) [inline app/screens/home/HomeScreen.tsx:343]: What is `open-in-new`?

**Review by davidleomay — COMMENTED** [review]
Add blockchain to buy/sell route data

File: app/i18n/es.json (hunk 2 above)
Comment (Kolibri1990) [reply app/i18n/es.json:208]: Yes. Just DE and EN (for all others) for the first step.

**Review by davidleomay — APPROVED** [review]

**Pull request merged**